Backpack/Calendar/Classes/BPKCalendarPriceLabelStyle.m

```diff
@@ -29,7 +29,7 @@ + (BPKCalendarPriceLabelStyle *)noData {
     static BPKCalendarPriceLabelStyle *style;
 
     dispatch_once(&onceToken, ^{
-      style = [[BPKCalendarPriceLabelStyle alloc] initWithTextColor:BPKColor.textSecondaryColor bold:NO];
+      style = [[BPKCalendarPriceLabelStyle alloc] initWithTextColor:BPKColor.textLinkColor bold:NO];
```

Comment:
Checking the snapshot but I think you have an option here because now many date are blue it seems like

Reply:
You should create a new style, probably `+ (BPKCalendarPriceLabelStyle *)iconData`

Reply:
ah this is a good spot, thanks. I'll fix